ack/utils/date_handler.py

```diff
@@ -69,6 +69,24 @@ def check_date_range_definition_conformity(start_date: date, end_date: date, dat
             raise DateDefinitionException("Report end date should be equal or ulterior to report start date.")
 
 
+def check_scheduled_parameters_definition_conformity(
+    scheduled_start_date: date, scheduled_end_date: date, frequency: str, date_range: str
+):
+
+    if not date_range:
```

Comment:
de ce que je vois c'est à peu près le même code que pour check_date_range_definition_conformity, y'a pas moyen d'appeler ça depuis cette nouvelle fonction, où on ferait les nouveaux tests en plus, pour éviter du code en double?

Reply:
The logic is not the same for a scheduled report and for a normal report. 
In a normal report, we shouldn't have both a date range and a couple of dates. 
In a scheduled report, it is needed. 

I could indeed use check_date_range_definition_conformity(scheduled_start_date, scheduled_end_date, None) after my else, but it seems to me that it is a "hack" just to avoid code deduplication that doesn't explain the logic behind the check. 

What do you think @AlexisVLRT ?

Reply:
It seems logical to me, thanks for the explanation